{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/production.py

```diff
@@ -12,11 +12,28 @@
 
 LOGGING['handlers']['local']['level'] = 'INFO'
 
+DICT_UPDATE_KEYS = ('JWT_AUTH',)
+
+FILE_STORAGE_BACKEND = {}
+
 CONFIG_FILE = get_env_setting('{{cookiecutter.repo_name|upper}}_CFG')
 with open(CONFIG_FILE, encoding='utf-8') as f:
-    config_from_yaml = yaml.load(f)
+    config_from_yaml = yaml.safe_load(f)
+
+    # Remove the items that should be used to update dicts, and apply them separately rather
+    # than pumping them into the local vars.
+    dict_updates = {key: config_from_yaml.pop(key, None) for key in DICT_UPDATE_KEYS}
+
+    for key, value in list(dict_updates.items()):
```

Comment:
Nit: what's with list()ing an iterator?  Couldn't this just be `for key, value in dict_updates.items():`?

Reply:
Yes! Changed it.